.drone.star

```diff
@@ -16,6 +16,9 @@ OC_CI_CLIENT = "owncloudci/client:latest"
 OC_CI_CORE = "owncloudci/core"
 OC_CI_DRONE_CANCEL_PREVIOUS_BUILDS = "owncloudci/drone-cancel-previous-builds"
 OC_CI_PHP = "owncloudci/php:%s"
+OC_OCIS = "owncloud/ocis:%s"
+OC_CI_WAIT_FOR = "owncloudci/wait-for:latest"
+OC_CI_NODEJS = "owncloudci/nodejs:16"
```

Comment:
It's not needed but other apps are following alphabetical order. I think we can put them in order with IDE's help.